### PR TITLE
Use new rules list structure

### PR DESCRIPTION
**Pull request by jeddy3**
> Which issue, if any, is this issue related to?

Ref: https://github.com/stylelint/stylelint/pull/6390

> Is there anything in the PR that needs further explanation?

Will need to update `package.json` on the next release.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
     "prettier": "^2.7.1",
     "remark": "^14.0.2",
     "remark-cli": "^11.0.0",
-    "stylelint": "https://github.com/stylelint/stylelint/tarball/14.13.0",
+    "stylelint": "file:../stylelint",
```
Comment (ybiquitous) [inline package.json:118]: Now it's possible to pass CI by using the HEAD version:

```suggestion
    "stylelint": "https://github.com/stylelint/stylelint/tarball/HEAD",
```

File: netlify.toml [hunk 2]
```diff
@@ -3,877 +3,789 @@
   publish = "build/"
 
 [[redirects]]
-  from = "/user-guide/configuration"
-  to = "/user-guide/configure"
+  from = "/user-guide/rules/list
+  to = "/user-guide/rules
```
Comment (ybiquitous) [inline netlify.toml:7]: Are quotes missing (also on other lines)?

```suggestion
  from = "/user-guide/rules/list"
  to = "/user-guide/rules"
```

Comment (jeddy3) [reply netlify.toml]: Good catch!. Fixed.

**Comment by jeddy3** [comment]
Preview LGTM. 👍

**Review by ybiquitous — APPROVED** [review]
Thank you, LGTM 👍🏼 

I'll update the `stylelint` version in `package.json` and merge this PR at the next release.

**Pull request merged**